ISSUE_TEMPLATE/testing.md

```diff
@@ -92,7 +92,32 @@ fedora-coreos-stream-generator -releases=https://fcos-builds.s3.amazonaws.com/pr
 - [ ] Once approved, merge it and push it to S3:
 
 ```
-aws s3 cp --acl=public-read streams/testing.json s3://fcos-builds/streams/testing.json --cache-control max-age=60
+aws s3 cp --acl public-read --cache-control 'max-age=60' streams/testing.json s3://fcos-builds/streams/testing.json
 ```
 
 - [ ] Verify the new version shows up on [the download page](https://getfedora.org/en/coreos/download/)
+
+## Rollout updates to existing hosts (updates metadata)
+
+From a checkout of this repo:
```

Comment:
since this step and the previous section both operate on checkouts of this repo and involve submitting PRs can you illustrate the value in opening two PRs vs just one PR?

Reply:
As we prefer. Those are two separate logical steps (as we are also performing them separately today and last time), but if it is more manageable we can bundle them in a single PR.

Reply:
Starting bundled for now to keep things easy SGTM, esp. while a lot of this stuff is mostly manual. As we automate, we need to keep in mind that they're separate and we need to retain the freedom to modify them independently if needed.